etcdctl/ctlv3/command/del_command.go

```diff
@@ -34,6 +35,8 @@ func NewDelCommand() *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&delPrefix, "prefix", false, "delete keys with matching prefix")
+	cmd.Flags().BoolVar(&delKVs, "deletedKVs", false, "get delted KVs")
```

Comment:
Something like `get-deleted` or `deleted-kvs` to follow other flag string patterns?